### PR TITLE
[DOCFIX] Update hive docs

### PR DESCRIPTION
**Pull request by wyykwan**

### CODE REVIEW

**Comment by wyykwan** [comment]
@cheng-chang ptal, thanks!

File: docs/en/compute/Hive.md [hunk 1]
```diff
@@ -337,7 +339,9 @@ You can run the following command in the Alluxio project directory:
 integration/checker/bin/alluxio-checker.sh hive -hiveurl [HIVE_URL]
 ```
 
-You can use `-h` to display helpful information about the command.
+You can use `-h` to display helpful information about the command, along with this detailed page on 
+[JDBC connection URLs](https://cwiki.apache.org/confluence/display/Hive/HiveServer2+Clients#HiveServer2Clients-JDBC) 
+within Hive.
```
Comment (madanadit) [inline docs/en/compute/Hive.md]: nit: remove "within Hive" and add "to specify the Hive URL"

Comment (wyykwan) [reply docs/en/compute/Hive.md]: i'll update it now!

**Comment by madanadit** [comment]
alluxio-bot, merge this please.

**Comment by madanadit** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.

**Pull request merged**